Navaids/Fixes_Non-UK/FIXES_France.txt

```diff
@@ -51,7 +51,6 @@ AMODO N048.25.10.000 E002.58.49.000
 AMONI N045.04.02.000 E004.49.57.000
 AMOPO N048.46.25.900 W003.13.56.400
 AMORO N046.30.00.000 E004.32.17.000
-AMPUM N049.32.26.600 E005.48.48.100
```

Comment:
How did you choose the order of preference to keep when conflict exists? In this one for instance, Belgium has it defined less precisely. It's probably not worth worrying about, just wondered.
![image](https://github.com/VATSIM-UK/UK-Sector-File/assets/61326713/1097f632-61eb-4d9d-b3d8-d5d99d3a640f)
Is it also worth also documenting this methodology described in the issue somewhere in the wiki so they don't get re-duplicated in future (if you haven't already)?

Reply:
`Belgium > France > Ireland > Netherlands > Norway > (Other); alphabetically.` is what I put in the issue, though why France has more detail than Belgium I haven't a clue - I'll do some more looking around to figure out which ones are precise and which aren't and figure something out 😄